feeds/wlan-ap/opensync/files/bin/flash-firmware

```diff
@@ -34,11 +34,14 @@ fi
 
 # Set the current version as inactive before the upgrade
 FW_VERSION=`sed -n 's/FW_IMAGE_ACTIVE:\(.*\)/\1/p' < /usr/opensync/.versions`
+FW_BACKUP=$(uci get system.tip.inactivefw)
 uci set system.tip.inactivefw="${FW_VERSION}"
 uci commit
 
 /sbin/sysupgrade $IMGFILE
 if [ "$?" != "0" ] ; then
+  uci set system.tip.inactivefw="${FW_BACKUP}"
+  uci commit
```

Comment:
you can do **uci commit system**. Since we are trying to commit system configuration file.